File: uri/area_do_circulo/1002.pas
program foo (Input, Output);
{foo}

function areadocirculo( var raio: Double ) : Double;
begin
    areadocirculo := 3.14159 * raio * raio;
end;

var 
    raio: Double;
    area: Double;

begin
    readLn (raio);
    area := areadocirculo( raio );
    writeln ('A=', area : 0 : 4);
end.

File: uri/area_do_circulo/1002.scala

object Main {

    def areadocirculo(raio: Double): Double = 3.14159 * raio * raio

    def main(args: Array[String]) {
        
        var raio = scala.io.StdIn.readDouble : Double
        var area = areadocirculo( raio ) 

        printf("A=%.4f\n", area) 
    }
}